generation/spade_gan/spade_gan.ipynb

```diff
@@ -0,0 +1,991 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    num_epochs = 100
</pre><p>The name of this variable needs to be changed to <code>max_epochs</code> so that our CICD system knows which variable to set to 2 for fast running. We can do this here and in the other notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
@virginiafdez you may have to check in ReviewNB where our comments are showing up. It's not clear from just Github where things are but they are in the right places if you look at the notebook in ReviewNB.

Reply:
@ericspod Sure! I looked at all the comments in ReviewNB. All the suggested changes should be there now in both tutorials.